lms/djangoapps/courseware/access.py

```diff
@@ -394,7 +394,6 @@ def can_see_in_catalog():
         """
         return (
             _has_catalog_visibility(courselike, CATALOG_VISIBILITY_CATALOG_AND_ABOUT)
-            or _has_staff_access_to_descriptor(user, courselike, courselike.id)
```

Comment:
instead of removing this, could you add a feature flag inside the _has_staff_access_to_descriptor method that allows to keep standard and custom behavior ?

Reply:
Yes I could add a feature flag, but I only want to change `can_see_in_catalog`.  I don't want to affect many other places where the staff could have permission.
Do you agree to add the feature flag only here?